app/views/aliases/_index.html.erb

```diff
@@ -1,2 +0,0 @@
-<%= render :partial => 'aliases/list', :locals => { :aliases => aliases } %>
```

Comment:
@manishyadavv Did you modify all these files using some script or manually?  What was the script/gem which you used?